web_transport/sdk/impl/web_transport_server_session.cc

```diff
@@ -224,5 +224,12 @@ void WebTransportServerSession::AcceptIncomingStream(
   }
 }
 
+void WebTransportServerSession::OnDatagramReceived(absl::string_view datagram) {
+  if (visitor_) {
```

Comment:
can we move all visitor calls to a dedicated thread?

Reply:
There is a task runner `event_runner_` designed for events. Client events are called in this task runner, but cleanup is not correctly handled for server side (https://github.com/open-webrtc-toolkit/owt-server/issues/1083), so events are still called in current thread to avoid crash. I'll move it to `event_runner_` in a separate change along with cleanup fixes.